.claude/commands/add-enhancement.md

```diff
@@ -45,22 +49,26 @@ Act as an experienced software architect to create a comprehensive enhancement p
      - **Proposal**: High-level description of the proposed solution
      - **Workflow Description**: Detailed workflow with actors and steps
      - **Mermaid Diagram**: Add a sequence diagram when applicable to visualize the workflow
-     - **Metadata**: Fill in creation-date with today's date (2025-10-21), set other fields to TBD
+     - **API Extensions**: Only fill this section if the user confirms the proposal adds/changes CRDs, admission and conversion webhooks, aggregated API servers, or finalizers. Otherwise, add a TODO comment asking the user to complete this section if applicable.
+     - **Implementation Details/Notes/Constraints**: Provide a high-level overview of the code changes required. Follow the guidance from the template: "While it is useful to go into the details of the code changes required, it is not necessary to show how the code will be rewritten in the enhancement." Keep it as an overview; the developer should fill in the specific implementation details.
+     - **Metadata**: Fill in creation-date with today's date (2025-10-28), tracking-link with the provided JIRA ticket URL, set other fields to TBD
 
 4. **Handle Unfilled Sections**: For sections that cannot be filled based on the input:
    - Add a clear comment like `<!-- TODO: This section needs to be filled in -->`
    - Provide guidance on what should be included
 
-5. **Concise Professional Writing**:
+5. **Writing Guidelines**:
    - Write in a clear, concise, professional manner
    - Focus on the essential information
    - Use bullet points and structured formatting
    - Avoid unnecessary verbosity
+   - **Line Length**: Keep lines in the generated enhancement at a maximum of 80 characters. It is acceptable to exceed by 10-15 characters when necessary (e.g., for URLs or code examples), but not more than that.
```

Comment:
What happens with a URL that exceeds 95 characters?

Reply:
let me test here :)